### ISSUE TITLE
lightning OOM during importing 10TB tpcc data

### ISSUE THREAD

**Issue by cyliu0** [pingcap/br#937]
Please answer these questions before submitting your issue. Thanks!

1. What did you do?
If possible, provide a recipe for reproducing the error.
Import 10TB tpcc data from S3

2. What did you expect to see?
Import successfully

3. What did you see instead?
```bash
$ ./bin/tidb-lightning -c tidb-lightning.toml
Verbose debug logs will be written to tidb-lightning.log

Killed
$ dmesg
[200987.553046] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0,global_oom,task_memcg=/user.slice/user-1000.slice/session-1.scope,task=tidb-lightning,pid=338843,uid=1000
[200987.553309] Out of memory: Killed process 338843 (tidb-lightning) total-vm:77675088kB, anon-rss:70417072kB, file-rss:0kB, shmem-rss:0kB, UID:1000 pgtables:140380kB oom_score_adj:0
[200990.771018] oom_reaper: reaped process 338843 (tidb-lightning), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB
```

4. What version of BR and TiDB/TiKV/PD are you using?

```bash
$ ./bin/tidb-lightning -V
Release Version: v5.0.0-nightly-10-ga3c7776
Git Commit Hash: a3c777615cdcfa25df30d2d97992156d00899e27
Git Branch: release-5.0
Go Version: go1.13
UTC Build Time: 2021-03-24 08:39:37
Race Enabled: false
```

5. Operation logs
   - Please upload `br.log` for BR if possible
   - Please upload `tidb-lightning.log` for TiDB-Lightning if possible
     [tidb-lightning.log](https://github.com/pingcap/br/files/6206122/tidb-lightning.log)
   - Please upload `tikv-importer.log` from TiKV-Importer if possible
   - Other interesting logs

6. Configuration of the cluster and the task
   - `tidb-lightning.toml` for TiDB-Lightning if possible
   - `tikv-importer.toml` for TiKV-Importer if possible
   - `topology.yml` if deployed by TiUP

7. Screenshot/exported-PDF of Grafana dashboard or metrics' graph in Prometheus if possible

**Comment by glorv** [comment]
Work around for release-5.0 in #928 but need to further investigate with master

**Comment by glorv** [comment]
closed by #753
